src/Plugin.php

```diff
@@ -2,24 +2,30 @@
 
 namespace OrcaServices\Heartbeat;
 
+use Cake\Console\CommandCollection;
 use Cake\Core\BasePlugin;
 use Cake\Core\PluginApplicationInterface;
+use Cake\Http\MiddlewareQueue;
+use Cake\Routing\RouteBuilder;
 
+/**
+ * Plugin
```

Comment:
Let's use the opportunity and add info, not just repeat it:
```suggestion
 * CakePHP Heartbeat Plugin
```